fxa-auth-db-server/docs/API.md

```diff
@@ -594,6 +609,55 @@ Content-Length: 2
     * Content-Type : 'application/json'
     * Body : `{"code":"InternalError","message":"...<message related to the error>..."}`
 
+## verifyToken : `POST /token/<tokenVerificationId>/verify`
+
+This method verifies sessionTokens.
+Note that it takes the tokenVerificationId
+specified when creating the token,
+NOT the tokenId.
+
+### Example
+
+```
+curl \
+    -v \
+    -X POST \
+    -H "Content-Type: application/json" \
+    -d '{"uid":"fdea27c8188b3d980a28917bc1399e47"}' \
+    http://localhost:8000/token/8e8c27b704dbf6a5dc556453c92e7506/verify
+```
+
+### Request
+
+* Method : POST
+* Path : `/token/<tokenVerificationId>/verify`
+    * tokenVerificationId : hex128
+* Params:
+    * uid : hex128
+
+### Response
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: 2
+
+{}
+```
+
+* Status Code : 200 OK
+    * Content-Type : 'application/json'
+    * Body : {}
+* Status Code : 404 Not Found
+    * Conditions: if no unverified tokens exist for `tokenVerificationId` and `uid`
+    * Content-Type : 'application/json'
+    * Body : `{"message":"Not Found"}`
```

Comment:
Does it return a 404?  IIUC the docs about `verifyToken` above suggest it will always return a `200 OK` with empty object body.